e2e/content-services/social/like.component.e2e.ts

```diff
@@ -0,0 +1,179 @@
+import { UploadActions } from '../../actions/ACS/upload.actions';
+import { NavigationBarPage } from '../../pages/adf/navigationBarPage';
+
+describe('Like component', () => {
+
+    const loginPage = new LoginPage();
+    const likePage = new LikePage();
+    const navigationBarPage = new NavigationBarPage();
+    const componentOwner = new AcsUserModel();
+    const componentVisitor = new AcsUserModel();
+    const secondComponentVisitor = new AcsUserModel();
+    const uploadActions = new UploadActions();
+
+    let emptyFile;
+
+    const emptyFileModel = new FileModel({
+        'name': resources.Files.ADF_DOCUMENTS.TXT_0B.file_name,
+        'location': resources.Files.ADF_DOCUMENTS.TXT_0B.file_location
+    });
+
+    beforeAll(async (done) => {
+        this.alfrescoJsApi = new AlfrescoApi({
+            provider: 'ECM',
+            hostEcm: TestConfig.adf.url
+        });
+
+        await this.alfrescoJsApi.login(TestConfig.adf.adminEmail, TestConfig.adf.adminPassword);
+
+        await this.alfrescoJsApi.core.peopleApi.addPerson(componentOwner);
+
+        await this.alfrescoJsApi.core.peopleApi.addPerson(componentVisitor);
+
+        await this.alfrescoJsApi.core.peopleApi.addPerson(secondComponentVisitor);
+
+        await this.alfrescoJsApi.login(componentOwner.id, componentOwner.password);
+
+        emptyFile = await uploadActions.uploadFile(this.alfrescoJsApi, emptyFileModel.location, emptyFileModel.name, '-my-');
+
+        await this.alfrescoJsApi.core.nodesApi.updateNode(emptyFile.entry.id,
+
+            {
+                permissions: {
+                    locallySet: [{
+                        authorityId: componentVisitor.getId(),
+                        name: 'Consumer',
+                        accessStatus: 'ALLOWED'
+                    }, {
+                        authorityId: secondComponentVisitor.getId(),
+                        name: 'Consumer',
+                        accessStatus: 'ALLOWED'
+                    }]
+                }
+            });
+
+        done();
+    });
+
+    afterAll(async (done) => {
+        await uploadActions.deleteFilesOrFolder(this.alfrescoJsApi, emptyFile.entry.id);
+        done();
+    });
+
+    describe('Social operations by users on their own components', () => {
+
+        beforeAll(() => {
+            loginPage.loginToContentServicesUsingUserModel(componentOwner);
+        });
+
+        beforeEach(async () => {
+            await navigationBarPage.clickSocialButton();
+        });
+
+        it('[C203006] Should be able to like and unlike their components but not rate them,', () => {
+            likePage.writeCustomNodeId(emptyFile.entry.id);
+            expect(likePage.getNodeIdFieldValue()).toEqual(emptyFile.entry.id);
+            likePage.clickLike();
+            expect(likePage.getLikeCounter()).toBe('1');
+            likePage.removeHoverFromLikeButton();
+            expect(likePage.getLikedIconColor()).toBe('rgba(33, 150, 243, 1)');
```

Comment:
I suggest saving these kind of strings('rgba(33, 150, 243, 1)') into variables, in this way we will also know which colour to expect and it will be easier to change the values if it will be the case.